src/pages/resultsView/enrichments/EnrichmentsUtil.tsx

```diff
@@ -563,6 +791,30 @@ export function pickGenericAssayEnrichmentProfiles(
     });
 }
 
+export function pickGenericAssayBinaryEnrichmentProfiles(
+    profiles: MolecularProfile[]
+) {
+    return profiles.filter(p => {
+        return (
+            p.molecularAlterationType ===
+                AlterationTypeConstants.GENERIC_ASSAY &&
+            p.datatype === DataTypeConstants.BINARY
+        );
+    });
+}
+
+export function pickGenericAssayCategoricalEnrichmentProfiles(
+    profiles: MolecularProfile[]
+) {
+    return profiles.filter(p => {
+        return (
+            p.molecularAlterationType ===
+                AlterationTypeConstants.GENERIC_ASSAY &&
+            p.datatype === DataTypeConstants.CATEGORICAL &&
+            p.patientLevel === false
```

Comment:
It's ok to have a patient-level profile for binary data, right? Sorry I don't remember this.